docs/compilers/CSharp/Compiler Breaking Changes - DotNet 10.md

```diff
@@ -357,3 +357,34 @@ class extension
 }
 ```
 
+## Extended partial members are now virtual
+
+***Introduced in Visual Studio 2022 version 17.15***
+
+We have fixed [an inconsistency](https://github.com/dotnet/roslyn/issues/77346)
+where interface members would not be implicitly `virtual` and `public` if they were also marked `partial`.
+Note that Visual Basic and other languages not supporting default interface members will start requiring to implement such `partial` interface members.
+
+This change has an effect only when language version is set to C# 14 or later.
+To keep the previous behavior in C# 14 and later, explicitly mark `partial` interface members as `sealed` and `private`.
+
+```cs
+((I)new C()).M(); // wrote 1 previously, writes 2 now
+
+partial interface I
+{
+    public partial void M();
+    public partial void M() // implicitly virtual now
```

Comment:
>// implicitly virtual now [](http://example.com/codeflow?start=28&length=25)

I think we should add examples targeting changes around accessibility, preferable when virtual-ness is not involved